specification/monitor/resource-manager/Microsoft.Insights/stable/2015-04-01/autoscale_API.json

```diff
@@ -803,6 +814,39 @@
           "type": "string"
         }
       }
+    },
+    "ScaleRuleMetricDimension": {
+      "required": [
+        "dimensionName",
+        "operator",
+        "values"
+      ],
+      "properties": {
+        "dimensionName": {
```

Comment:
@LeoVannelli Could you confirm service's behavior? I tried this swagger spec and added one dimension into the profile. Then I used `get` method to retrieve the information of this rule.
![image](https://user-images.githubusercontent.com/49134743/86756497-02dd8500-c075-11ea-8fd1-5dff9c8802be.png)
The server's response is not consistent with this swagger spec. It should be `dimensionName` instead of `DimensionName` and same with `operator` and `values`. It would cause problem in python sdk and Azure CLI.

Reply:
The casing is implemented server side is wrong (code artifact of data contracts classes), but has been in use for a long time, these changes to the swagger are to fix that it was missed previously. Changing this server side may be possible but may also introduce a break for existing callers it is not clear it is safe to change this this late.